styleguide/_themes/global/scss/fonts.scss

```diff
@@ -159,3 +172,24 @@ i {
 .copyright {
   font-size: .75em;
 }
+
+blockquote {
+  border-left: 0;
+  font-size: 1.75rem;
```

Comment:
Seems to be pretty consistent with what we talked about in Wheaties. I didn't get any objections on it, so I think that's enough of a consensus to move forward with.